fuzzing/nxt_basic_fuzz.c

```diff
@@ -51,19 +51,19 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 void
 nxt_base64_fuzz(const u_char *data, size_t size)
 {
-    u_char  *input;
-    u_char  buf[256];
+    ssize_t  ret;
+    u_char   buf[256];
 
     /*
-     * Adding unnecessary NUL bytes to resolve buffer-overflow read and
-     * use-of-uninitialized-value in nxt_base64_decode nxt_string.c:833:21.
+     * Do a validation check on base64 data before decoding.
      */
 
-    input = nxt_zalloc(size + 1);
-    nxt_memcpy(input, data, size);
+    ret = nxt_base64_decode(NULL, (u_char *)data, size);
+    if(ret == NXT_ERROR) {
```

Comment:
Space between the `if` and opening `(` please.

Reply:
Missed it, that's why a formatter is necessary.